cmd/restic/cmd_prune.go

```diff
@@ -73,6 +77,10 @@ func verifyPruneOptions(opts *PruneOptions) error {
 		}
 		opts.MaxRepackBytes = uint64(size)
 	}
+	if opts.UnsafeNoSpaceRecovery != "" {
+		// prevent repacking data to make sure users cannot get stuck.
+		opts.MaxRepackBytes = 1
```

Comment:
Now that #3591 is merged this can be changed to `= 0` :-)

Reply:
And now it's 0.